src/pages/PasswordResets.tsx

```diff
@@ -22,7 +22,7 @@ export default function PasswordResets() {
     async function validateToken() {
       const isValid = await ApiPasswordResets.get(token || '')
       if (!isValid) {
-        toast.error('Password reset has expired, please try again', {
+        toast.error(t('expiredToken'), {
```

Comment:
Now that a translation string is used here, it looks like the dependency array on line 32 needs to be updated to `[token, navigate, t]` to pass the linter.

Reply:
Ohh good catch, thanks!